modules/helm-initializer/main.tf

```diff
@@ -26,7 +26,18 @@ data "template_file" "tiller_rbac" {
   }
 }
 
+data "external" "tiller_status" {
+  program = [
+    "bash",
+    "-c",
+    "REPLICAS=$$(kubectl get deploy tiller-deploy -n ${var.tiller_namespace} -o jsonpath='{.status.readyReplicas}'); jq -n --arg replicas \"$$REPLICAS\" '{readyReplicas:$$replicas}'"]
```

Comment:
What advantage do we get from using `jq` here and returning json vs returning readyReplicas as a bare string? AFAIK trigger doesn't need JSON as input.

Reply:
Ok, thanks.